.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
         include: # Run windows tests on only one python version
           - os: windows-latest
             python-version: '3.11'
```

Comment:
should this be 3.12 as well?

Reply:
Maybe another PR. Not really a priority atm